driver/csiplugin/controllerserver.go

```diff
@@ -518,6 +517,25 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		return nil, status.Error(codes.Aborted, fmt.Sprintf("volume creation already in process : %v", scaleVol.VolName))
 	}
 
+	jobDetails := cs.Driver.snapjobstatusmap[scaleVol.VolName]
+	if jobDetails.jobStatus == SNAP_JOB_RUNNING {
+		glog.Errorf("volume:[%v] -  Snapshot copy request in progress for snapshot: %s.", scaleVol.VolName, snapIdMembers.SnapName)
+		return nil, status.Error(codes.Aborted, fmt.Sprintf("Snapshot copy request in progress for snapshot: %s", snapIdMembers.SnapName))
+	} else if jobDetails.jobStatus == SNAP_JOB_FAILED {
+		glog.Errorf("volume:[%v] -  Snapshot copy job had failed for snapshot %s", scaleVol.VolName, snapIdMembers.SnapName)
+		return nil, status.Error(codes.Internal, fmt.Sprintf("SSnapshot copy job had failed for snapshot: %s", snapIdMembers.SnapName))
```

Comment:
error strings should not be capitalized and there is typo SSnapshot

Reply:
Done